### ISSUE TITLE
english, french, chinese, vietnamese translation needed

### ISSUE THREAD

**Issue by holema** [H2-invent/jitsi-admin#45]
Pleas add a translation in english and french

**Comment by anholzmann** [comment]
We need support for translating the app into english and french

**Comment by resbkesier** [comment]
Hi my name is Renato, I am a Brazilian developer and I really liked the project, although I am unable to implement it in my infrastructure, I would like to better understand the project and its particularities in order to contribute in some way.

**Comment by resbkesier** [comment]
Hi guys, my name is Renato, I'm a Brazilian programmer and I'm doing some tests with your application, I would like to help translate if you are interested.

I still haven't been able to make it work here, I'm having errors with multiple redirects, I don't know if it's the application or if I configured the keycloak client incorrectly

have you found this error?

let's talk about the translation, it would be interesting for me to be able to help the project in some way

thanks

**Comment by xdaniel2063** [comment]
Hi there, I would really love to change the background of the Join-page to a colour instead of a changing photo.

**Comment by xdaniel2063** [comment]
Another Thing is to change the text of the emails. It would be great to add additional links and individual texts. Maybe the agenda field can be used for this, but I don't know if html is allowed there?

**Comment by xdaniel2063** [comment]
Another feature I'm looking for is to transfer the moderation rights to someone else in my team?

**Comment by itrich** [comment]
Will you use an collanorative translation tool, e.g. like Crowdin? I'll have a look whether I can find a volunteer for French 😀

**Comment by AlexanderGabriel** [comment]
Also englisch kann ich übernehmen. Sind ja bald ein paar Feiertage :)
Wie genau funktioniert denn das in den xlf-Dateien?
resname und source lassen und target mit den __ übersetzen?

**Comment by AlexanderGabriel** [comment]
Wie stell' ich den Jitsiadmin denn um auf Englisch?

**Comment by itrich** [comment]
Ich sehe, dass ihr auf Crowdin setzt. Darf sich die Community beteiligen?

**Comment by holema** [comment]
@itrich klar, es ist als public project gemarkt und die anfrage als Open-Source Projekt läuft.

Die Umstellung vom jitsi-admin funktioniert durch die verwendung des präfix /en/ bzw /fr/ (https://jitsi-admin.de/en/room/dashboard)

Die Buttons zum Sprachen wechseln haben wir noch deaktiviert da wir ja noch keine Übersetzung hatten.

**Comment by holema** [comment]
@itrich @AlexanderGabriel Das ist der Public Link https://crwd.in/jitsi-admin

**Comment by itrich** [comment]
Klasse! Auf welchem Branch wollt ihr denn die Übersetzung pflegen? Eigentlich macht das Crowdin automatisch, aber ich habe gesehen, dass ihr mehrere Branches in Crowdin synchronisiert?

**Comment by holema** [comment]
@itrich es ist der master branch
den translaation Branch müssen wir wieder löschen
Da hat crowdin sich ertsmal vertan :)

**Comment by AlexanderGabriel** [comment]
Wollen wir Meeting und Konferenz vielleicht vereinheitlichen in Meeting?

**Comment by faust64** [comment]
Hi,

Looking at translations, I found that translation with ID `Fh87GFQ` and `9OpdI6M` should probably be merged into one.
```
<trans-unit id="Fh87GFQ" resname="Sie wurden zu dem Serientermin {name} hinzugefügt. Sie erhalten vor jedem Termin eine E-Mail mit dem Link zur Einwahl.">
<trans-unit id="9OpdI6M" resname="Sie wurden zu dem Serientermin {name} hinzugefügt. Sie erhalten vor jedem Termin eine Email mit dem Link zur Einwahl.">
```

Same goes for IDs `maPhtKz` and `THWcDaJ`
```
<trans-unit id="maPhtKz" resname="Die Veranstaltung {name} wurde in einen Serientermin überführt. Sie erhalten vor jedem Termin eine Email mit dem Link zur Einwahl.">
<trans-unit id="THWcDaJ" resname="Die Veranstaltung {name} wurde in einen Serientermin überführt. Sie erhalten vor jedem Termin eine E-Mail mit dem Link zur Einwahl.">
```

Maybe even `1YU8isJ` and `ahaO0xj`:
```
<trans-unit id="1YU8isJ" resname="Alle Serienelement der Serie bearbeiten">
<trans-unit id="ahaO0xj" resname="Alle Serienelemente bearbeiten">
```

Regards.

**Comment by infra-monkey** [comment]
Hi,
Does this require some specialized tool or it can be manual?

**Comment by holema** [comment]
Hello @infra-monkey,
you can translate it directly at crowdin. https://crowdin.com/project/jitsi-admin
Then it is directly pushed into the project here at github.

**Comment by infra-monkey** [comment]
@holema Thank you.
I submitted some changes to the english language. Do I need to do something else to create a PR from it?
